dali/pipeline/data/tensor.h

```diff
@@ -46,6 +46,57 @@ class Tensor : public Buffer<Backend> {
   inline ~Tensor() override = default;
 
 
+  /** @defgroup ContiguousAccessorMembers Contiguous accessor member functions
+   * Public buffer access to Tensor underlying memory - Tensors are always contiguous.
+   * @{
+   */
+
+  /**
+   * @brief Returns a typed pointer to the underlying storage. If the
+   * buffer has not been allocated because it does not yet have a type,
```

Comment:
```suggestion
   * tensor has not been allocated because it does not yet have a type,
```

Reply:
As this is the Tensor API I would stick to using `tensor` in the docs.

Reply:
I just copied the old Doxygen verbatim from tensor, I can adjust if you want.

Reply:
I would appreciate that.

Reply:
All removed, will resolve stale comments.